README.md

```diff
@@ -1,12 +1,26 @@
 # Vulcanomicon
 
-The Vulcan Academy's operating system
+The Vulcan Academy's operating system.
 
-1. The goal of the game is to create a learning organization
-1. All players must unanimously agree to all rule changes
-1. Proposals may add, amend or repeal a rule
-1. All rules should be logically self-consistent 
+1. The goal of the game is to create a learning organization.
+1. All players must unanimously agree to all rule changes.
+1. Proposals may add, amend or repeal a rule.
+1. All rules should be logically self-consistent. 
+
+## Reality
+1. Our understanding of __Reality__ is throttled by our ability to percieve it. 
+    * We use logic to describe our understanding of __Reality__.
+1. __Reality__ is exclusively inhabited by __Information__, and patterns thereof.
+    * __Information__ contained within __Empirical Reality__ inhabits 3 spatial dimensions and 1 monotonically increasing temporal dimension.
+1. __Information__ may interact with and modify other __Information__. This is an __Action__.
 
 ## Lexicon
 
-agent: any system that is best characterized as acting according to beliefs
\ No newline at end of file
+* __Reality__: The set of everything that exists or does not exist.
+* __Information__: What is conveyed by a particuar arrangement of things.
+* __Empirical Reality__: __Reality__ that is verifiable by observation rather than theory or pure logic.
+* __Action__: The interaction between __Information__.
+* __Decision Function__: Given __Reality__ _R_ and __Actions__ _A_ = { _a_<sub>0</sub>, ... }, we define __Decision Function__ _d_ such that _d_(_R_) = { _a_<sub>k</sub>, ... }.
+* __Agent__: A distinguishable set of __Information__ that has the ability to perform __Actions__. __Agents__ choose which __Action__ to perform based on a __Decision Function__.
+    * An __Agent__ inhabiting __Empirical Reality__ evaluates its __Decision Function__ at every infintesimally small time-dimension increment.
+* __Rational Agent__: an __Agent__ who's __Decision Function__ operates according to beliefs.
```

Comment:
Not sure Rational is the right qualifier here since an agent could theoretically act according to its irrational beliefs.

Reply:
Doesn't that conflate the rationality of the agent with the rationality of the belief?

Reply:
To me a Decision Function is the same as beliefs in this context. Not sure if we need this definition then as the above definition of agent pretty much covers it. Ie, I perceive "Agents choose which actions to perform based on a decision function" to be essentially equal to "Agents choose which actions to perform based on their beliefs"

Reply:
But I could define a decision function that does not operate on the concept of beliefs. For example an agent could choose its actions based on an FSM, or even a random choice.

My thinking was that a belief based system (claim + truth value) is a specialization of a decision function. Kind of like the decision function being an interface which the belief based approach implements.

Reply:
Sure, that makes sense. I really think we want to add a definition of belief then as well

Reply:
I agree, I think @macterra would provide the best definition there 😃.

Reply:
Hmm github emoji's really don't provide much of a middle ground for emotion, i was going for more of a relaxed closed-mouth smiley.

Reply:
☺️